src/System.Reflection.DispatchProxy/src/System.Reflection.DispatchProxy.csproj

```diff
@@ -11,15 +11,25 @@
     <AssemblyVersion Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETFramework'">4.0.6.0</AssemblyVersion>
     <AssemblyVersion Condition="'$(IsPackable)' == 'true' and $([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETFramework'">4.0.7.0</AssemblyVersion>
     <PackageValidationBaseLineVersion>4.7.1</PackageValidationBaseLineVersion>
+    <IsPlaceholderTargetFramework Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETCoreApp'">true</IsPlaceholderTargetFramework>
+    <EnableDefaultItems>false</EnableDefaultItems>
   </PropertyGroup>
 
-  <ItemGroup Condition="$(TargetFramework) == 'netstandard2.0'">
-    <Compile Remove="System\Reflection\DispatchProxyGenerator.cs" />
-    <Compile Remove="System\Reflection\Emit\IgnoreAccessChecksToAttributeBuilder.cs" />
+  <ItemGroup Condition="'$(TargetFramework)' != 'netstandard2.1'">
```

Comment:
nit: this will be included for `netcoreapp3.1`.  You might consider the check like the one here here: https://github.com/dotnet/maintenance-packages/blob/d4261b1112437567ca4841e0c251a0ec914e7213/src/Microsoft.Bcl.HashCode/src/Microsoft.Bcl.HashCode.csproj#L14